test/encryption-decryption.test.ts

```diff
@@ -32,13 +32,13 @@ describe('aes-256-gcm', () => {
     }
   });
 
-  Object.values(CipherStrategy).forEach((strategy) => {
-    Object.values(SerializationFormat).forEach((version) => {
+  Object.values([CipherStrategy.AES_GCM]).forEach((strategy) => {
+    Object.values([SerializationFormat.latest_version]).forEach((version) => {
```

Comment:
I think revert the above two? This was just to reduce the number of tests, right?

Reply:
Yes, will fix it, soon. Thanks